src/frontend/src/components/Wizard/WizardStep.tsx

```diff
@@ -0,0 +1,65 @@
+import Button from "../base/Button/Button";
+import { wizardStyle } from "./Wizard.styles";
+
+import ArrowLeftIcon from "assets/icons/arrow-left.svg?react";
+
+interface WizardStep {
+    question: string;
+    description?: string;
+    buttonText?: string;
+    disableBack?: boolean;
+    children: React.ReactNode;
+    handleBack: () => void;
+    handleNext: () => void;
+}
+
+const WizardStep = ({
+    question,
+    description = undefined,
+    buttonText = "Continue",
+    disableBack = false,
+    handleBack,
+    handleNext,
+    children,
+}: WizardStep) => {
+    const {
+        stepContainer,
+        questionContainer,
+        questionText,
+        descriptionText,
+        buttonContainer,
+    } = wizardStyle();
+
+    return (
+        <div className={stepContainer()}>
+            <div className={questionContainer()}>
+                <h3 className={questionText()}>{question}</h3>
+                {!!description && (
+                    <p className={descriptionText()}>{description}</p>
+                )}
+            </div>
+            {children}
+            <div className={buttonContainer()}>
+                <Button
+                    variant="outline"
+                    size="large"
+                    isDisabled={disableBack}
+                    leftIcon={
+                        <ArrowLeftIcon className="font-normal h-[14px] w-[14px] text-black" />
```

Comment:
I agree it's tough to tell the "previous" button is disabled on the first step, what do you think about a quick styling change like below?:
```suggestion
                         <ArrowLeftIcon className={`font-normal h-[14px] w-[14px] ${disableBack ? "fill-gray-300" : "text-black"}`} />
```
<img width="470" height="412" alt="Screenshot 2025-10-08 at 2 01 59 PM" src="https://github.com/user-attachments/assets/a9724ae1-d2c8-43e3-93ae-644e7ffad857" />